templates/swift/Example/PROJECT/Base.lproj/Main.storyboard

```diff
@@ -1,21 +1,26 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?>
-<document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="6211" systemVersion="14A298i" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" initialViewController="vXZ-lx-hvc">
+<?xml version="1.0" encoding="UTF-8"?>
+<document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="13771" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" colorMatched="YES" initialViewController="vXZ-lx-hvc">
+    <device id="retina4_7" orientation="portrait">
+        <adaptation id="fullscreen"/>
+    </device>
     <dependencies>
-        <plugIn identifier="com.apple.InterfaceBuilder.IBCocoaTouchPlugin" version="6204"/>
+        <deployment identifier="iOS"/>
+        <plugIn identifier="com.apple.InterfaceBuilder.IBCocoaTouchPlugin" version="13772"/>
+        <capability name="documents saved in the Xcode 8 format" minToolsVersion="8.0"/>
     </dependencies>
     <scenes>
         <!--View Controller-->
         <scene sceneID="ufC-wZ-h7g">
             <objects>
-                <viewController id="vXZ-lx-hvc" customClass="ViewController" customModuleProvider="target" sceneMemberID="viewController">
+                <viewController id="vXZ-lx-hvc" customClass="ViewController" customModule="PROJECT_Example" customModuleProvider="target" sceneMemberID="viewController">
```

Comment:
Is PROJECT expanded by the ruby script with the user selection?

Reply:
Thanks @jugutier :)

I did not use any scripts to do these changes. I opened the project with Xcode 9.2 and made the updates.

Reply:
Yep
https://github.com/CocoaPods/pod-template/blob/9ecc1048b63698fece03ed7b8a6ac12015091f0d/setup/ProjectManipulator.rb#L21-L27

Reply:
Try to use it locally with this command:

pod lib create --template-url='file:///path/to/pod-template' MyPod

and see if it expands EXAMPLE into MyPod

Reply:
I confirmed that it expands correctly. Output:

```
<viewController id="vXZ-lx-hvc" customClass="ViewController" customModule="MyPod_Example" customModuleProvider="target" sceneMemberID="viewController">
```

In IB:
<img width="262" alt="screen shot 2017-12-28 at 8 10 10 am" src="https://user-images.githubusercontent.com/2844185/34411751-dcfb522c-eba6-11e7-9b2e-3a1281d001e8.png">